--- Slune-1.0.12rc1/slune~	2006-09-23 23:16:32.000000000 +0200
+++ Slune-1.0.12rc1/slune	2006-09-23 23:15:59.000000000 +0200
@@ -36,10 +36,8 @@
   sys.path.insert(0, APPDIR)
 
 import slune.globdef as globdef, soya
-soya.init(width = globdef.SCREEN_WIDTH, height = globdef.SCREEN_HEIGHT, fullscreen = globdef.FULLSCREEN, sound = 1, sound_doppler_factor = 0.005, sound_reference_distance = 3.0)
-#soya.init(width = globdef.SCREEN_WIDTH, height = globdef.SCREEN_HEIGHT, fullscreen = globdef.FULLSCREEN)
+soya.init(width = globdef.SCREEN_WIDTH, height = globdef.SCREEN_HEIGHT, fullscreen = globdef.FULLSCREEN)
 soya.set_quality(globdef.QUALITY)
-soya.set_sound_volume(globdef.SOUND_VOLUME)
 
 LEVEL        = ""
 PARRAIN_HOST = None
